### ISSUE TITLE
Wrong assertion for floating point numbers

### ISSUE THREAD

**Issue by TKolesnikova** [codeborne/xls-test#2]
When trying to assert that xls file contains float number "21.11":

`assertThat(xls, containsText("21.11"));`

I'm receiving AssertionErorr:

```
java.lang.AssertionError: 
Expected: a XLS containing text "21.11"
     but: was "...\Показания+на+31.12.2020+00_00_00.xls"
Показание, кВт*ч	
General21
```	
	
But my xls file ([Показания+на+31.12.2020+00_00_00.xlsx](https://github.com/codeborne/xls-test/files/5981888/%2B.%2B31.12.2020%2B00_00_00.xlsx)) contains expected number and assert should be passed.

**Comment by asolntsev** [comment]
@TKolesnikova Should be fixed by PR https://github.com/codeborne/xls-test/pull/3.

**Comment by asolntsev** [comment]
I have released the fix as `xls-test:1.5.0`

@TKolesnikova  @FrameBassman
